SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -1366,13 +1372,10 @@ void CFlowOutput::WriteForcesBreakdown(CConfig *config, CGeometry *geometry, CSo
 
     }
 
-
     /*--- Write file name with extension ---*/
 
-    string filename = config->GetBreakdown_FileName();
-    strcpy (cstr, filename.data());
```

Comment:
Ggood thing imo to replace those `strcpy` operations. I guess we could get rid of all of these (on another day in another PR ;) ), or is there any reason to not prefer directly working with strings?

Reply:
Well, I see you just removed some more `strcpy`'s 👍

Reply:
In some places they are needed because the constructor of CConfig for example takes a char array.
Some are there because SPRINTF is used to modify the name, that would have to be replaced probably by stringstreams.
I tried to remove some obvious ones that were 100% not needed.